p2p/net/swarm/swarm_dial.go

```diff
@@ -416,7 +417,7 @@ func (s *Swarm) dialAddrs(ctx context.Context, d *conn.Dialer, p peer.ID, remote
 	for i := 0; i < len(remoteAddrs); i++ {
 		select {
 		case exitErr = <-errs: //
-			log.Debug(exitErr)
```

Comment:
:-1:  this happens all the time-- it will be super annoying for users. use notice and grep?